stages/org.osbuild.vagrant

```diff
@@ -2,11 +2,139 @@
 import json
 import os
 import subprocess
+import random
 import sys
+import uuid
 
 import osbuild.api
 
 
+# https://github.com/hashicorp/vagrant/blob/241810b28af77fbb008c6863ce8a180919d8b86b/website/content/docs/providers/virtualbox/boxes.mdx#raw-contents
+BOX_XML = """<?xml version="1.0"?>
+<Envelope ovf:version="1.0" xml:lang="en-US" xmlns="http://schemas.dmtf.org/ovf/envelope/1" xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:vbox="http://www.virtualbox.org/ovf/machine">
+  <References>
+    <File ovf:id="file1" ovf:href="{DISK_NAME}"/>
+  </References>
+  <DiskSection>
+    <Info>List of the virtual disks used in the package</Info>
+    <Disk ovf:capacity="{DISK_CAPACITY}" ovf:diskId="vmdisk1" ovf:fileRef="file1" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized" vbox:uuid="{DISK_UUID}"/>
+  </DiskSection>
+  <NetworkSection>
+    <Info>Logical networks used in the package</Info>
+    <Network ovf:name="NAT">
+      <Description>Logical network used by this appliance.</Description>
+    </Network>
+  </NetworkSection>
+  <VirtualSystem ovf:id="packer-virtualbox-UNIX">
+    <Info>A virtual machine</Info>
+    <OperatingSystemSection ovf:id="100">
+      <Info>The kind of installed guest operating system</Info>
+      <Description>Linux26_64</Description>
```

Comment:
What about extracting at least some hardcoded bits to stage options? e.g. `Linux26_64` feels quite outdated already.